isis/tests/FunctionalTestsCnetCombinePt.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+
+#include <QHash>
+#include <QSet>
+#include <QString>
+
+#include <boost/foreach.hpp>
+
+#include "Pvl.h"
+#include "Fixtures.h"
+#include "gmock/gmock.h"
+#include "UserInterface.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetcombinept.xml").expanded();
+
+QSet<QString> combineMerges(QHash<QString, QSet<QString>> &mergeLog, QSet<QString> newMerges) {
+  QSet<QString> combinedMerges(newMerges);
+  BOOST_FOREACH ( QString pointId, newMerges ) {
+    // Recursively append everything previously merged into this point
+    if (mergeLog.contains(pointId)) {
+      combinedMerges.unite(combineMerges(mergeLog, mergeLog.take(pointId)));
+    }
+  }
+  return combinedMerges;
+}
```

Comment:
Any particular reason this is defined in both `cnetcombinept.cpp` and the test suite?

Reply:
Nevermind, looking at an older version 😬

Reply:
This was a test I used during development that is not exposed anymore so I yanked it.